adv_manhole/models/__init__.py

```diff
@@ -1,10 +1,11 @@
 from adv_manhole.models.monodepth2.monodepth2_model import MonoDepth2
 from adv_manhole.models.supergradients.supergradients_model import SuperGradientsModel
+from adv_manhole.models.depth_hints.depth_hints_model import DepthHints
 from typing import Optional
 from enum import Enum
 
 # Create ENUM for model types
-ModelType = Enum("ModelType", ["MDE", "SS"])
+ModelType = Enum("ModelType", ["MDE", "SS", "DH", "MD"])
```

Comment:
@Sekigahara Please use the same "MDE" for both DepthHints and ManyDepth since the model are Monocular Depth Estimations